application/backend/app/schemas/source.py

```diff
@@ -121,3 +123,41 @@ class ImagesFolderSourceConfig(BaseIDNameModel):
 ]
 
 SourceAdapter: TypeAdapter[Source] = TypeAdapter(Source)
+
+# ---------------------------------
+# Creation Schemas (POST requests)
+# ---------------------------------
+# These schemas inherit from HasID first to override the required ID field with an auto-generated one (if absent) via
+# MRO (Method Resolution Order).
+
+
+class DisconnectedSourceConfigCreate(HasID, DisconnectedSourceConfig):
+    pass
+
+
+class WebcamSourceConfigCreate(HasID, WebcamSourceConfig):
+    pass
+
+
+class IPCameraSourceConfigCreate(HasID, IPCameraSourceConfig):
+    pass
+
+
+class VideoFileSourceConfigCreate(HasID, VideoFileSourceConfig):
+    pass
+
+
+class ImagesFolderSourceConfigCreate(HasID, ImagesFolderSourceConfig):
+    pass
+
+
+SourceCreate = Annotated[
+    DisconnectedSourceConfigCreate
```

Comment:
Can you exclude the "disconnected" source from the list of sources that can be created?